src/app/View.elm

```diff
@@ -33,6 +34,17 @@ firstPlayer players =
             "No Players Found"
 
 
-tableView : Html Msg
-tableView =
-    Html.map TableMsg (Table.view Table.init)
+tableView : List Player -> Dice.Cup -> Html Msg
+tableView players cup =
+    let
+        -- we have to fake ids until we have them
+        idAndPlayerTo3Tuple id { playerName, diceCount } =
+            ( id, playerName, diceCount )
+
+        ids =
+            List.range 1 <| List.length players
+
+        playersTuples =
+            List.map2 idAndPlayerTo3Tuple ids players
+    in
+        Table.view Types.TableAction <| Table.initWithPlayersAndCup playersTuples cup
```

Comment:
By the way, I don't take the prescriptive advice that a view function should always be a certain way or anything like that. `view` is just a function that renders the view as far as I can see. In `TEA`, it's prescribed that it should be a certain form, but it's only a recommendation. There's nothing that says it can't just be a *value* even (ie `thingView : Html Msg` is perfectly acceptable if you don't need state). It's only the actual `view` function itself that gets hooked into the `Html.program` function that has to be `view : Model -> Html Msg`, if you see what I mean?